contributionVerifier.js

```diff
@@ -1,9 +1,20 @@
 const requestp = require('./requestAsPromise');
+const {githubRequest,getReadmeContents} = require('./githubApi');
+
+const clabotToken = process.env.GITHUB_ACCESS_TOKEN;
 
 const contributorArrayVerifier = (contributors) =>
   (committers) =>
     Promise.resolve(committers.filter(c => contributors.indexOf(c) === -1));
 
+const configFileFromGithubUrlVerifier = (contributorListGithubUrl) =>
+  (committers) => githubRequest({
+      url: contributorListGithubUrl,
+      method: 'GET'
+    },clabotToken)
+    .then((body) => githubRequest(getReadmeContents(body),clabotToken))
```

Comment:
@maoo I don't think this line was needed? the above request fetches the contributor list, do no need to fetch a README?